pkg/kv/kvclient/rangefeed/rangefeed_external_test.go

```diff
@@ -1645,3 +1646,210 @@ func (c *channelSink) Send(e *kvpb.RangeFeedEvent) error {
 		return c.ctx.Err()
 	}
 }
+
+// TestRangeFeedMetadataManualSplit
+func TestRangeFeedMetadataManualSplit(t *testing.T) {
+	defer leaktest.AfterTest(t)()
+	defer log.Scope(t).Close(t)
+
+	testutils.RunValues(t, "feed_type", procTypes, func(t *testing.T, s feedProcessorType) {
+		ctx := context.Background()
+		settings := cluster.MakeTestingClusterSettings()
+		// We must enable desired scheduler settings before we start cluster,
+		// otherwise we will trigger processor restarts later and this test can't
+		// handle duplicated events.
+		kvserver.RangeFeedUseScheduler.Override(ctx, &settings.SV, s.useScheduler)
+		srv, _, db := serverutils.StartServer(t, base.TestServerArgs{
+			Settings: settings,
+		})
+		defer srv.Stopper().Stop(ctx)
+		ts := srv.ApplicationLayer()
+
+		scratchKey := append(ts.Codec().TenantPrefix(), keys.ScratchRangeMin...)
+		_, _, err := srv.SplitRange(scratchKey)
+		require.NoError(t, err)
+		scratchKey = scratchKey[:len(scratchKey):len(scratchKey)]
+		mkKey := func(k string) roachpb.Key {
+			return encoding.EncodeStringAscending(scratchKey, k)
+		}
+
+		sp := roachpb.Span{
+			Key:    scratchKey,
+			EndKey: scratchKey.PrefixEnd(),
+		}
+
+		// Enable rangefeeds, otherwise the thing will retry until they are enabled.
+		for _, l := range []serverutils.ApplicationLayerInterface{ts, srv.SystemLayer()} {
+			// Whether rangefeeds are enabled is a property of both the
+			// storage layer and the application layer.
```

Comment:
I don't think this should be true anymore. I think you _should just be able to set it on the storage/system layer.  Can we set this on the settings object we pass to testserver directly?

Reply:
done.